VerbrauchZeitspanne/README.md

```diff
@@ -43,8 +44,8 @@ Die Statusvariablen/Kategorien werden automatisch angelegt. Das Löschen einzeln
 
 Name        | Typ     | Beschreibung
 ----------- | ------- | ----------------
-Start-Datum | Integer | Start-Datum für den Verbrauch
-End-Datum   | Integer | End-Datum für den Verbrauch
+Start-Datum | Integer | Start-Datum für den Verbrauch (Sekunden werden nicht beachtet)
```

Comment:
Macht es Sinn dies nur "Start" und Ende" zu nennen? Und dann in der Beschreibung Start-Datum/Start-Zeit?

Reply:
Sieht besser aus.
Bezieht sich das auch auf die Variablen Namen?